src/main/java/mate/academy/spring/exception/UserNotSavedException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.spring.exception;
+
+public class UserNotSavedException extends Exception {
```

Comment:
```suggestion
public class DataProcessingException extends RuntimeException {
```
let's make the exception more generic, we will be able to reuse it on dao layer
and why have you decided to make it a checked exception?

Reply:
Good question. Is it possible somehow that we will handle this exception and try to save something one more time, for example?